src/bun.js/node/types.zig

```diff
@@ -98,6 +98,19 @@ pub fn Maybe(comptime ResultType: type) type {
             };
         }
 
+        pub inline fn initErr(e: Syscall.Error) Maybe(ReturnType) {
```

Comment:
is initErr and initResult worth it over typing .{ .err = e } directly? its fine but ive always just inlined it (though would be legendary if the decl literals proposal was implemented)